plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/helpers.ts

```diff
@@ -30,6 +30,10 @@ export async function getOctokitOptions(options: {
 }): Promise<OctokitOptions> {
   const { integrations, credentialsProvider, repoUrl, token } = options;
   const { owner, repo, host } = parseRepoUrl(repoUrl, integrations);
+  const requestOptions = {
+    // set timeout to 60 seconds
+    timeout: 600,
```

Comment:
I'm guessing the comment should read "600 seconds" ?

Reply:
I think it's ms, which means I need to change the value too. I'll follow up.

Thanks for that